src/daemon.py

```diff
@@ -103,11 +104,15 @@ def __execute_yara(self, job: JobId, files: List[str]) -> None:
                     )
             except yara.Error:
                 logging.exception(f"Yara failed to check file {sample}")
+                num_errors += 1
             except FileNotFoundError:
                 logging.exception(
                     f"Failed to open file for yara check: {sample}"
                 )
+                num_errors += 1
+        logging.exception(f"Error files: {num_errors}")
```

Comment:
1) This is not related to an exception, so `logging.error` at most
2) This is being logged unconditionally (there should be no errors in logs when there are no errors)
3) As a general rule, in case of logging, prefer `logging.func("asdf %s", param)`  instead of `logging.func(f"asdf {param}")`
4) finally, this log is unncessary - all errors are logged already with more details anyway 

```suggestion
        logging.exception(f"Error files: {num_errors}")
```